docs/future_improvements.md

```diff
@@ -1,3 +1,76 @@
+# Open Auctions
```

Comment:
It is probably worth mentioning that it should be implemented as a separate set of scripts or combined into universal ones that will support closed and open auctions.